macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -5,12 +5,10 @@
 import FlutterMacOS
 import Foundation
 
-import path_provider_macos
 import sqflite
 import sqlite3_flutter_libs
 
 func RegisterGeneratedPlugins(registry: FlutterPluginRegistry) {
-  PathProviderPlugin.register(with: registry.registrar(forPlugin: "PathProviderPlugin"))
```

Comment:
In my opinion, this part shouldn't be in this PR.
I suggest you to add the missing code and use `--amend` command to update this PR :D

Reply:
Thanks, I've restored the original file.